testData/solutions/win32.json5

```diff
@@ -2742,5 +2742,117 @@
         "start": [],
         "stop": [],
         "isInstalled": [] // always reported as not installed. Should be changed once GPII-1998 is fixed
+    },
+
+    "net.gpii.browserExtension": {
+        "name": "UIO+ Browser Extension",
+        "contexts": {
+            "OS": [{
+                    "id": "win32",
+                    "version": ">=5.0"
+            }]
+        },
+        "settingsHandlers": {
+            "chromeconf": {
+                "type": "gpii.settingsHandlers.webSockets",
+                "options": {
+                    "path": "net.gpii.browserExtension"
+                },
+                "capabilities": [
+                    "http://registry\\.gpii\\.net/common/lineSpace",
```

Comment:
I had a problem with a couple of the preferences in the nested transformations not being inferred. It could be that I had something wrong in those transformations at the time.